### ISSUE TITLE
Undo delete removes editable and movable properties

### ISSUE THREAD

**Issue by maribethb** [google/blockly#6793]
**Describe the bug**

Undoing a deletion removes the movable and editable properties from blocks.

**To Reproduce**

Load the following XML into the playground:

```
<xml xmlns="https://developers.google.com/blockly/xml">
  <block type="text_print" id="wlc#l2D-,6mSLKY=E$VL" x="-76" y="-112">
    <next>
      <block type="text_print" id="T9cdR}mwDn3Xl!*g|mw8" movable="false">
        <comment pinned="true" h="32" w="88.5333251953125">Immovable</comment>
      </block>
    </next>
  </block>
  <block type="text_print" id="ZgFlirwB{~}%l/BZ+j?R" x="47" y="-118"/>
</xml>
```

Delete the immovable block.
Undo.
The block is now movable.

**Expected behavior**

Undo restores the former state.

**Additional context**

My assumption is that this is due to JSON serialization for default blocks. The JSON serializer does not serialize the editable, movable, and deletable properties of blocks unless you install a plugin and in that case it serializes them separately.

The BlockDelete event uses JSON serialization instead of XML serialization to restore the deleted block, so these properties are lost.

This also happens even if you are using the serialize-disabled-properties plugin, because the additional serializer is not updated when the block is created.

**Comment by BeksOmega** [comment]
> This also happens even if you are using the serialize-disabled-properties plugin, because the additional serializer is not updated when the block is created.

Yeah the plugin serializers are all workspace-level, but the copy-paste and create/delete events use the block serializer directly, so the custom serialize-disabled-properties serializer isn't getting hit.

**Comment by BeksOmega** [comment]
Fixed by #6847
